containers/version.go

```diff
@@ -0,0 +1,23 @@
+package containers
+
+import (
+	"context"
+	"strings"
+
+	"dagger.io/dagger"
+)
+
+// GetPackageJSONVersion gets the "version" field from package.json in the 'src' directory.
+func GetPackageJSONVersion(ctx context.Context, d *dagger.Client, src *dagger.Directory) (string, error) {
```

Comment:
Do we really have to go through Dagger for something like this?

Reply:
I prefer to use dagger wherever possible, especially in scenarios like this because it both caches the result and ensures consistency.